Git Formats/Mailmap.sublime-syntax

```diff
@@ -0,0 +1,18 @@
+%YAML 1.2
+---
+# Syntax based on documentation here:
+# https://www.git-scm.com/docs/git-check-mailmap#_mapping_authors
+
+name: Mailmap
```

Comment:
Both the file name and the name here should start with "Git" as the other files.

- Git Mailmap.sublime-syntax
- `name: Git Mailmap`

Syntaxes are addressed via file name only. Hence the name should be meaningful enough to avoid conflicts with other packages.

Reply:
I'm not strictly opposed, but I don't think it's necessary.

- Searching DDG for `mailmap` returns lots of entries for Git-related stuff.
- Searching DDG for `mailmap -git` _still_ returns mostly Git-related stuff.
- There are no packages named Mailmap.
- There are [hardly any packages](https://packagecontrol.io/search/mail) with "mail" anywhere in the name

Reply:
All kind of files in this package start with Git. Wouldn't break that guideline.